taskcluster/ci/swift/kind.yml

```diff
@@ -45,7 +45,7 @@ tasks:
         - ["source", "taskcluster/scripts/toolchain/setup-libs-ios.sh"]
       commands:
         - ["taskcluster/scripts/build-and-test-swift.sh"]
-        - [ "cd", "build/" ]
+        - [ "cd", "artifacts/" ]
```

Comment:
Shouldn't the build still happen in build/, and we'd just change the `tar` command below to generate `../artifacts/swift-components.tar.xz`?

Reply:
Makes sense, I just updated the script to do that.